test/test_matmul_cuda.py

```diff
@@ -39,8 +38,8 @@ def tearDown(self):
         torch.backends.cuda.matmul.allow_tf32 = True
         super(self.__class__, self).tearDown()
 
-    @onlyCUDA
```

Comment:
same comment.
any reason to remove this? This UT is only for GPU both CUDA and ROCm.